src/app/components/ScrollLock.tsx

```diff
@@ -39,14 +39,19 @@ const fadeIn = keyframes`
 
 interface OverlayProps extends HTMLAttributes<HTMLDivElement> {
   className?: string;
+  onClick?: () => void;
   mobileOnly?: boolean;
   zIndex?: number;
 }
 
 // tslint:disable-next-line:variable-name
 export const Overlay = styled(({ mobileOnly, zIndex, ...props}: OverlayProps) => {
   useDisableContentTabbing();
-  return <div {...props} />;
```

Comment:
you should leave this spread, because the props include everything that you can put on a div

Reply:
the `onClick` was giving a weird console error, refactoring it the way I did fixed it

Reply:
or `className`, one of those

Reply:
nvm something from master fixed the issue, changing it back